src/utils/context.jsx

```diff
@@ -26,10 +29,12 @@ const context = function context(Component, params) {
       }
       let slide = 0;
       if (this.props.params && "slide" in this.props.params) {
-        slide = this.props.params.slide;
+        slide = Number(this.props.params.slide);
```

Comment:
@knowbody @kenwheeler 1. When I added a slide with a string id, forward navigation did fail with Number(this.props.params.slide) and succeeded when I changed this line back to this.props.params slide. 2. But there are two warnings about the type of slide, so its contextType needs to become PropTypes.oneOfType([PropTypes.number,PropTypes.string]) here and also in src/deck.jsx